charts/posthog/templates/_snippet-initContainers-wait-for-migrations.tpl

```diff
@@ -0,0 +1,51 @@
+{{/* Common initContainers-wait-for-migrations definition */}}
+{{- define "_snippet-initContainers-wait-for-migrations" }}
+- name: wait-for-migrations
+  image: {{ include "posthog.image.fullPath" . }}
+  command:
+    - /bin/sh
+    - -c
+    - >
+        until (python manage.py migrate --check);
```

Comment:
I can't remember what the outcome was here, should we also be waiting on migration_clickhouse and anything else that might need to be run?